Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1996,16 +2066,21 @@ void IMixedRealityPointerHandler.OnPointerClicked(MixedRealityPointerEventData e
         /// <inheritdoc/>
         void IMixedRealityTouchHandler.OnTouchStarted(HandTrackingInputEventData eventData)
         {
+            if (currentPointer != null)
```

Comment:
Can you add a comment for why this no-ops if currentPointer is null (i.e. it has to do with the event ordering between OnPointerDown and OnTouchStarted). It's the difference between near/far interactions right?